<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0" rowspan="2" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_media">
          <property name="minimumSize">
           <size>
            <width>426</width>
            <height>230</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>Просмотр изображения</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_4">
          <property name="minimumSize">
           <size>
            <width>426</width>
            <height>241</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>241</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QFrame" name="frame_3">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>30</y>
             <width>741</width>
             <height>88</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>426</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>88</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>11</x>
              <y>21</y>
              <width>721</width>
              <height>55</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Типы объектов&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="lineEdit_types">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>304</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maxLength">
                  <number>256</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_setTypes">
                 <property name="text">
                  <string>Сохранить</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QFrame" name="frame_2">
           <property name="geometry">
            <rect>
             <x>-10</x>
             <y>140</y>
             <width>754</width>
             <height>99</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>426</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>130</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>240</x>
              <y>20</y>
              <width>271</width>
              <height>71</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="pushButton_import">
               <property name="minimumSize">
                <size>
                 <width>201</width>
                 <height>23</height>
                </size>
               </property>
               <property name="text">
                <string>Выбрать изображения для предпросмотра</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_detect">
               <property name="minimumSize">
                <size>
                 <width>201</width>
                 <height>23</height>
                </size>
               </property>
               <property name="text">
                <string>Распознать объекты</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
